docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -1,90 +1,201 @@
+
+The script assumes that you will run it as a non-`root` user, but the shell in
+the container you created earlier is running as root. Remove `sudo` from the
+start of the script so you can run it on your container. It should look
+something like this:
+
+```text
+bash -c "$(curl -fsSL https://TENANT_SUBDOMAIN.teleport.sh/scripts/SCRIPT_NAME/install-node.sh)" 
+```
+
+Leave the script running in your shell. Teleport will run in the foreground. You
+will see a message in the Web UI that your new server connected successfully:
+
+![Connected
+successfully](../../../img/cloud/getting-started/successfully-connected@2x.png)
+
+Click **Next**.
 
-Install client libraries:
+Confirm that you will authenticate as the `root` user and click **Next**:
 
-<Tabs>
-  <TabItem label="MacOS">
-(!docs/pages/includes/enterprise/install-macos-ent.mdx!)
-  </TabItem>
-  <TabItem label="Windows">
-(!docs/pages/includes/install-windows.mdx!)
-  </TabItem>
+![Choose your user](../../../img/cloud/getting-started/set-up-access@2x.png)
 
-  <TabItem label="Linux">
-    ```code
-    $ curl -O https://cdn.teleport.dev/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    # verify signature 
-    $ echo "$(curl  https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256)" | sha256sum --check
-    $ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    $ cd teleport-ent
-    $ sudo ./install
-    ```
-  </TabItem>
-</Tabs>
+From there, you can test the connection and access the server. Click **Start
+Session** to connect:
 
-Login into Teleport and test the connection:
+![Start session](../../../img/cloud/getting-started/test-connection@2x.png)
+
+<Notice type="tip">
+
+If you get an error connecting to your container, interrupt the `teleport`
+process in your container's shell with **Ctrl-C**, then run `teleport start` to
+restart the process.
+
+</Notice>
+
+You should see a terminal prompt in your browser. Your Teleport Enterprise Cloud
+account routes your SSH connection through the Teleport Proxy Service, which
+connects to your container via a reverse tunnel. 
+
+Reverse tunnels like this allow Teleport to manage access to resources like
+Kubernetes clusters and remote desktops, even if these resources are running on
+private networks.
+
+Execute some commands in your terminal. When you have finished, run `exit` to
+end your session.
+
+In the **Manage Access** view, click **Finish**, then **Browse existing
+resources** to visit the main view of the Teleport Web UI.
+
+## Step 3/4. Play back your session
+
+As Teleport Enterprise Cloud proxies SSH connections to registered servers, it
+records the commands that users execute during their sessions so operators can
+play them back later to investigate issues.
+
+In the Teleport Web UI, click the **Activity** tab in the sidebar on the left of
+the screen, then click **Ression Recordings**. You will see your session from
```

Comment:
@ptgott please correct this spelling

Reply:
Yes https://github.com/gravitational/teleport/pull/21265

Reply:
To avoid conflicts you'll likely want to just cherry-pick the commit fixing it.